django/db/models/functions/text.py

```diff
@@ -150,6 +150,22 @@ class LTrim(Transform):
     lookup_name = 'ltrim'
 
 
+class MD5(Transform):
+    function = 'MD5'
+    lookup_name = 'md5'
+
+    def as_oracle(self, compiler, connection, **extra_context):
+        return super().as_sql(
+            compiler,
+            connection,
+            template=(
+                "LOWER(RAWTOHEX(STANDARD_HASH(UTL_I18N.STRING_TO_RAW("
+                "%(expressions)s, 'AL32UTF8'), 'MD5')))"
+            ),
+            **extra_context
```

Comment:
Add a trailing comma. (Supported by Python 3.5+ — [flake8-commas](https://pypi.org/project/flake8-commas/) is a neat extension...)